src/Slider.tsx

```diff
@@ -361,7 +369,7 @@ export const Slider = forwardRef<SliderTypes.API, PropsWithChildren<Settings>>((
                     </div>
                 ))}
             </div>
-            { !hideNavigationButtons && (
+            { showNavigationButtons && (
                 <>
                     <PreviousButton onClick={scrollToPreviousSlide} isHidden={!prevArrowVisible} direction={orientation}/>
```

Comment:
We could also consider doing conditional rendering right here, so we don't have to introduce multiple ways to keep track of visibility. Something like:

```
prevArrowVisible && <PreviousButton onClick={scrollToPreviousSlide} isHidden={!prevArrowVisible} direction={orientation}/>
```

Reply:
Yeah you're right, we should only do this if both controls should not be rendered.